twopaneview/src/TwoPaneView.tsx

```diff
@@ -151,48 +126,7 @@ export class TwoPaneView extends Component<Props, State> {
     );
   }
 
-  getEntireSize() {
-    var size = {
-      width: this.state.dims.width,
-      height: this.state.dims.height,
-    };
-    return size;
-  }
-
-  getTopSize() {
-    let topHeight = this.state.dims.height/2;
-    var size = {
-      width: this.state.dims.width,
-      height: topHeight,
-    }
-    return size;
-  }
-
-  getBottomSize() {
-    let topHeight = this.state.dims.height/2; 
-    var size = {
-      width: this.state.dims.width,
-      height: topHeight,
-    }
-    return size;
-  }
-
-  getLeftSize() {
-    let leftWidth = this.state.dims.width/2; 
-    var size = {
-      width: leftWidth,
-      height: this.state.dims.height,
-    }
-    return size;
-  }
-
-  getRightSize() {
-    let rightWidth = this.state.dims.width/2; 
-    var size = {
-      width: rightWidth,
-      height: this.state.dims.height,
-    }
-    return size;
+  isHorizontalOrientation() {
+    return (this.state.rotation === 'rotation0' || this.state.rotation === 'rotation180');
```

Comment:
Again an enum on rotation would solve the need to know about the rotation strings

Reply:
+1